lib/kuber_hex/gateways/authorize_net.ex

```diff
@@ -33,68 +59,222 @@ defmodule Gringotts.Gateways.AuthorizeNet do
   #----------------making requests to gateway
 
   @doc """
-    function to authorize the merchant using merchant name
-    and transactionKey
+  Authenticate a merchant
+
+  The functions takes the name and transactionKey from the config and verifies if 
```

Comment:
So it verifies if the given configuration is correct by asking AuthorizeNet? What's the use case?

Reply:
It's just for verifying the merchant authentications, so that before making other requests, the merchant can check whether the configurations are correct or not. But, planning to remove this because this method is not being provided in the base methods.